src/styles/index.less

```diff
@@ -10,3 +10,7 @@ body {
 #root {
   height: 100%;
 }
+
+a {
+  color: unset;
+}
```

Comment:
What is this for?

Reply:
Basically quick fix for unstyling `a` tags - `<Link>` wraps elements in `<a>` and gives them sometimes unwanted styles, wanna expand on this later but didn't want too many changes in this PR